set.mm

```diff
@@ -810491,6 +810542,678 @@ as an isomorphism in the category of graphs (something like "GraphIsom
+      UVDAVNZBVOKZVSZCVOKZVSZUDLZVNZUELZUWOKUWMKZAUWQUWKKZUBZMZUEUWLVPZJZUDVTZJ
+      ZUUNUUQUVSUVOQUWMUEUDUWKABCDUVDFUWHUWKVGZUWMVGZVQUXEUUNUUQRZUVRUVOHUVAUXH
+      UVPUVAIZJZUVRUVOUXJUVRJZUVEUVFUVNUXKUVEUVQUVDIZUXKDUVDUVPAUXHDUVDAWHZUXIU
+      VRUXEUUNUXMUUQUWJUXMUXDDUVDAWASWBZWCUXJUVPDIZUVRUXIUXOUXHBEUVPDFWDTSWEUVR
+      UVEUXLPZUXJUXPUVCUVQUVCUVQUVDWFWGTWIUXHUVFUXIUVRUXHDUVDEAUXNUXEUUNUUQVEWE
+      WCUXKUVNUWEUUSUVQULZUVJNZGUVLOZUNZUXJUXTUVRUXIUXHUXTUXIUXOUUQJZUVPEMZEUVP
+      ULZUFLZNZUFBUMKZOZUNZJUXHUXTQZUFUYFBUVPDEFUYFVGWJUYHUYAUYIUYBUYAUYIQUYGUY
+      BUXTUYAUXHUYBUWEUXSUVPEAWKWLWRUYAUYGUYIUYAUYEUYIUFUYFUYAUYDUYFIZJZUYEUXHU
+      XTUYKUYEUXHRUXSUWEUYKUYEUXHUXSUYAUYJUYEUXHUXSQQUXHUYJUYEUYAUXSUXHUYJUYEUY
+      AUXSQZQZUXHUYJJZUYDUGLZUWKKZMZUGUWLOZUYMUXHUYJUYRUUNUXEUYJUYRPZUUQUULUYSU
+      UMUGUYDBUWKUXFWMSWNWOUYNUYQUYMUGUWLUXHUYJUYOUWLIZUYQUYMQZUXEUUNUUQUYJUYTV
+      UAQZQZUWJUXDUUNUUQVUCQQZUWJUXCVUDUDUWJUWPUXBVUDUWJUWPJZUYJUUNUUQUXBVUBVUE
+      UYJUUNUUQUXBVUBQQVUEUYJUUNRZUUQUYTUXBVUAVUFUUQUYTUXBVUAQVUFUUQUYTRZUXBUYO
+      UWOKZUWMKZAUYPUBZMZVUAUYTVUFUXBVUKQUUQUXAVUKUEUYOUWLUWQUYOMZUWRVUIUWTVUJU
+      WQUYOUWMUWOWPVULUWSUYPAUWQUYOUWKWKWQWSWTUPVUGVUKUYQUYMVUGVUKUYQRUYEUYCUYP
+      NZUYLUYQVUGUYEVUMPVUKUYDUYPUYCXAUPVUGVUKVUMUYLQUYQVUGVUKJZVUMUYAUXSVUNVUM
+      UYARZUXRUXQVUINGVUIUVLUVJVUIUXQXAVUNVUMVUIUVLIZUYAVUGVUPVUKVUFUYTVUPUUQVU
+      FUWMXBZUYTVUHUWNIVUPUUNVUEVUQUYJUUMVUQUULUWMCUXGXCTUPVUFUWLUWNUYOUWOVUEUY
+      JUWLUWNUWOWHZUUNUWPVURUWJUWLUWNUWOWATWBXDUWMCVUHUXGXEXFXGSWBVUOUXQVUJVUIV
+      UOUXQAUYCUBZVUJVUOADXHZUUQUXORZVUSUXQMVUOVUTUUQUXOJZJZVVAVUNUYAVVCVUMVUNV
+      UTUYAVVBVUGVUTVUKVUFUUQVUTUYTVUEUYJVUTUUNUWJVUTUWPDUVDAXSZSWBWBSUXOUUQXIV
+      HXGVUTUUQUXOXJXKDEUVPAXLXMVUMVUNVUSVUJNUYAUYCUYPAXNWNXOVUGVUKVUMUYAXPXQUS
+      XRXTYAXRYBXRYCXRYDVRYEYFYGYHYIYKYJYLYFYGYMXRYIYNYOYPYQYPSUVRUVNUXTPZUXJVV
+      EUVCUVQUVCUVQMZUVHUWEUVMUXSUVCUVQUUSYRVVFUVKUXRGUVLVVFUVIUXQUVJUVCUVQUUSY
+      SUUAVAUUCWGTWIUUDYJYIUUEUUFXRYPYFUVTUVOPUURGUVLCUVCUVDUUSUWHUWIWJYTUUPVUT
+      UVADNZUWAUVSPUUQUUOVUTUUNUUOUWJVUTABCDUVDFUWHUUGVVDXMTVVGUUQBEDFUUBYTHDUV
+      AUVCAUUHUUIUUJUUK $.
+  $}
+
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Local isomorphisms of graphs
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+
+  This section is about local isomorphisms of graphs, which are a
+  generalization of isomorphisms of graphs, i.e., every isomorphism between two
+  graphs is also a local isomorphisms between these graphs, see ~ uhgrimgrlim .
+
+  This definition is according to a chat in mathoverflow
+  ( ~ https://mathoverflow.net/questions/491133/locally-isomorphic-graphs ):
+  roughly speaking, it restricts the correspondence of two graphs to their
+  neighborhoods.
+
+  Additionally, a binary relation ` ~=lgr ` is defined (see ~ df-grlic ) which
+  is true for two graphs iff there is a local isomorphisms between these
```

Comment:
Non-blocking
typo:
there are local isomorphisms
or
there is a local isomorphism